### DISCUSSION TITLE
Unable to connect to repository

### DISCUSSION THREAD

**Discussion by EvilAaron** [croneter/PlexKodiConnect#1922] [Q&A] [answered]
Just Installed Kodi on windows 11 to try to setup PKC.

Setup repository with zip file from wiki and HTTP croneter.github.io method.

Unable to connect to repository to install regardless method. 
![image](https://user-images.githubusercontent.com/7889822/219939148-4db87353-f8e9-4d76-a525-b2ede50eade8.png)

Am I doing something wrong.

**Comment by croneter** [comment, marked as answer]
Yes you are 😄  - works for everyone else. Did you pick the correct repository for you Kodi version? (Matrix Nexus etc.). Logs would also help, see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by EvilAaron** [comment]
Ahhh thank-you for the reply, gave me the hint needed to find where I went wrong. What I get for trying to setup stuff hours past my bedtime lol.

Used the circled repository download, missing the unmarked ones at top lol, working now, sorry for the stupid post ;)

![Screenshot 2023-02-20 091736](https://user-images.githubusercontent.com/7889822/220168438-958811ca-98fc-4f3f-a3f6-37f44f4630ce.png)

**Comment by stormsh** [comment]
Hello there,
why does the latest PlexKodiConnect-3.10.0.zip not show up in the repository? 
https://croneter.github.io/pkc-source/
Newest is 3.7.0 (which is a beta).

Thx 4 F1.

**Reply by croneter** [reply]
@stormsh: The PKC repository is its own Add-on that you need to install FIRST. Once you've done that, navigate to the PKC repository. THEN install the actual PKC add-on. You'll see the latest version just fine 😉. See https://github.com/croneter/PlexKodiConnect/wiki/Installation for details. 

TLDR;
You need 2 install TWO add-ons: the PKC repository. Then after that PKC itself

**Reply by stormsh** [reply]
Yes. I know that it's two Addons. The PKC repository is installed (see screenshot).
The newest version should show in the PKC repository, but it doesn't.
The Matrix Beta is the latest which is 3.7... something.


![installed](https://github.com/croneter/PlexKodiConnect/assets/4153852/4936a35b-c025-4f8c-a089-66f15cd4ab86)

**Reply by croneter** [reply]
@stormsh Just checking: On the screenshot you shared, install "repo....Nexus.STABLE" (the last one). Then install PKC from the PKC repo. Can you share a screenshot of the available add-ons in the PKC repo?

**Reply by in2dp** [reply]
I had the same problem. downloaded the stable version from the icon bottom above, and was shown the same error msg. version Croneter 1.0.2.

**Reply by in2dp** [reply]
showed: All repositiries, Kodi add-on repository, Plexkodiconnect addons, and Plexkodiconnet beta addons for Jodi 20 nexus (3.0.0)

**Reply by in2dp** [reply]
managed to get the repository open and install the video add-on, however, noting happens to prompt me for Plex login. and going into the configuration and choose from a list of servers, clicked and nothing happens. also, I am getting an error message that ask me to look into the log, which I could not find.

**Reply by croneter** [reply]
@in2dp please follow this guide to get a DEBUG log: https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Reply by stormsh** [reply]
> @stormsh Just checking: On the screenshot you shared, install "repo....Nexus.STABLE" (the last one). Then install PKC from the PKC repo. Can you share a screenshot of the available add-ons in the PKC repo?

Yes I can.

![01](https://github.com/croneter/PlexKodiConnect/assets/4153852/35be4398-2780-4512-8748-3165cce93f4c)
![02](https://github.com/croneter/PlexKodiConnect/assets/4153852/75b3da84-42bf-460c-be9a-2bcab40920ad)
![03](https://github.com/croneter/PlexKodiConnect/assets/4153852/6cd327a3-bebb-4527-bc4d-5a47832a9f34)

**Reply by croneter** [reply]
@stormsh all as expected! The crucial last step is missing, can you please share a screenshot where you now go into the PKC repository you just installed in order to install PlexKodiConnect itself?

**Reply by stormsh** [reply]
Something seems to be missing. I can not find the pkc repo. When I go to Settings - Addons - Install from repository  I just get the Kodi Add-on repo. I cannot choose the pkc repo. 

And I don´t see an Omega pkc addon. Do I use the latest nexus stable or beta?


![04](https://github.com/croneter/PlexKodiConnect/assets/4153852/beb1b755-6f92-41f3-886a-411b549f7510)
![05](https://github.com/croneter/PlexKodiConnect/assets/4153852/94349c74-4f5c-46cb-b7ce-39078282bdc7)

**Comment by croneter** [comment]
@stormsh 
Here, install the repository "repository.plexkodiconnect....Nexus.STABLE" - yes, Nexus, it will also work for Omega. You haven't done that yet. 
![02](https://github.com/croneter/PlexKodiConnect/assets/4153852/75b3da84-42bf-460c-be9a-2bcab40920ad)

Go back here, choose "Install from repository", then the PlexKodiConnect repo will show up. 
![04](https://github.com/croneter/PlexKodiConnect/assets/4153852/beb1b755-6f92-41f3-886a-411b549f7510)

**Comment by stormsh** [comment]
Finally. I think I got it?! 3.10.2 Took me long enough... sry. And thank you very much.

![06](https://github.com/croneter/PlexKodiConnect/assets/4153852/fc2e8792-328b-45b2-9ee8-9d1a62035ae5)
